### PR TITLE
Add LogEntry + Shape

### PR DESCRIPTION
**Pull request by PHochmann**

### CODE REVIEW

**Comment by kragall** [comment]
Hi, so I checked with the current version of the Clearing House and saw that the originally requested structure doesn't fit anymore. So I wondered if it's possible to adjust the changes to fit this structure:

- **timestamp**: This is the timestamp recording the date when this message was logged by the CH
- **transaction_id**: This is basically a counter with a unique (unique in the database) number for each stored message.
- **process_id**: This is a string that describes the process under which the message is logged. Think of it as the label of the folder under which the message has been filed together with other messages.
- **document_id**: This is a unique (unique in the process) id in the form of a uuid for the stored message
- **client_id**: This is the unique id of the client that sent the stored message taken from the DAT the client sent with the original LogMessage
- **server_id**: This is the unique id of the server that stored message in form of the id of private key
- **hash**: This information is used for integrity protection in the CH
- **signature**: This contains a cryptographic signature of the data
- **payload**: This optional field contains the payload of the original LogMessage

Please note that in the original request LogEntry contained a LogMessage. This is not the case anymore.

**Comment by PHochmann** [comment]
> Hi, so I checked with the current version of the Clearing House and saw that the originally requested structure doesn't fit anymore. So I wondered if it's possible to adjust the changes to fit this structure:
>     * **payload**: This optional field contains the payload of the original LogMessage

Is this payload a string or another Data Type property, or does it point to another object? I can't find any information about what a LogMessage's payload looks like.

**Comment by kragall** [comment]
The payload does not reference another entity, so it's probably best to use a string

**Comment by PHochmann** [comment]
Edits are under taxonomies/Message.ttl @kragall Please have a look

**Review by kragall — CHANGES_REQUESTED** [review]

File: taxonomies/Message.ttl [hunk 1]
```diff
@@ -385,6 +385,51 @@ ids:LogMessage a owl:Class;
...
     rdfs:comment "Log Message which can be used to transfer logs e.g., to the clearing house."@en.
 
+ids:LogEntry a owl:Class;
+    rdfs:label "Log Entry"@en ;
+    rdfs:comment "Contains the metadata of a LogMessage"@en .
+
+ids:transactionID a owl:DatatypeProperty ;
+    rdfs:domain ids:LogEntry ;
+    rdfs:range xsd:unsignedInt ;
+    rdfs:comment "Links a LogEntry (Metadata) to a transactionID"@en .
```
Comment (JohannesLipp) [inline taxonomies/Message.ttl:395]: @PHochmann I suggest to add `rdfs:label` to all newly added properties. That would basically be readable versions of their URIs, like `transactionID` -> `transaction id/ID`

File: taxonomies/Message.ttl [hunk 2]
```diff
@@ -385,6 +385,51 @@ ids:LogMessage a owl:Class;
...
+    rdfs:comment "Links a LogEntry (Metadata) to a processID"@en .
+
+ids:documentID a owl:DatatypeProperty ;
+    rdfs:domain ids:LogEntry ;
+    rdfs:range xsd:string ; # UUID?
+    rdfs:comment "Links a LogEntry (Metadata) to a documentID"@en .
+
+# Timestamp given by dct:created
+
+ids:hasHash a owl:DatatypeProperty ;
```
Comment (JohannesLipp) [inline taxonomies/Message.ttl:409]: Be more consistent with naming conventions? The other new properties do not have a verb notation (clientID, payload, signature, ...), but this one has (**has**Hash). Easy solution = remove the "has" part from this line?

```suggestion
ids:hash a owl:DatatypeProperty ;
```

**Review by JohannesLipp — CHANGES_REQUESTED** [review]

File: taxonomies/Message.ttl [hunk 3]
```diff
@@ -385,6 +385,60 @@ ids:LogMessage a owl:Class;
     rdfs:label "Log Message"@en ;
     rdfs:comment "Log Message which can be used to transfer logs e.g., to the clearing house."@en.
 
+ids:LogEntry a owl:Class;
+    rdfs:label "Log Entry"@en ;
+    rdfs:label "Log Entry"@en ;
```
Comment (kragall) [inline taxonomies/Message.ttl:390]: 389 and 390 are the same. Is that a c&p error?

**Comment by PHochmann** [comment]
All requested changes are now resolved, feel free to review and merge. Thanks!

**Review by JohannesLipp — APPROVED** [review]

**Pull request merged**